lib/ecto/parameterized_type.ex

```diff
@@ -191,7 +201,12 @@ defmodule Ecto.ParameterizedType do
       @doc false
       def equal?(term1, term2, _params), do: term1 == term2
 
-      defoverridable embed_as: 2, equal?: 3
+      # @doc false
+      def inspect(parameterized_type) do
+        inspect(parameterized_type)
```

Comment:
```suggestion
        Kernel.inspect(parameterized_type)
```

And, as per the above, it should only receive the params. :)